src/main/java/net/levelz/mixin/server/world/ServerWorldMixin.java

```diff
@@ -0,0 +1,40 @@
+package net.levelz.mixin.server.world;
+
+import net.levelz.access.PlayerStatsManagerAccess;
+import net.levelz.network.PlayerStatsServerPacket;
+import net.minecraft.entity.Entity;
+import net.minecraft.server.network.ServerPlayerEntity;
+import net.minecraft.server.world.ServerWorld;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Mixin(ServerWorld.class)
+public class ServerWorldMixin {
```

Comment:
why do we need a sync here?
I think the serverplayerentity already does that

Reply:
It doesn't take effect when cross dimension or death, maybe the client cleaned up the data.

Reply:
Then the mixin json has to get updated too